### PR TITLE
SDK-1769 Introduced the Newrelic extension.

### PR DESCRIPTION
**Pull request by pavelmaksimov25**
## Describe your changes.
NewRelic support for the SprykerCI environment.

## Issue ticket link
Ticket: [{Ticket Link}](https://spryker.atlassian.net/browse/SDK-1769).

## Checklist before requesting a review
- [ ] I have added ticket link to the PR description.
- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by DmytroKlymanSpryker — DISMISSED** [review]

File: infrastructure/sdk.sprykerci.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+RUN git config --add --system safe.directory /project
+
+########################################
+# New Relic Extension
+# It's already in the core image.
+########################################
+ARG NEWRELIC_ENABLED
+ENV NEWRELIC_ENABLED ${NEWRELIC_ENABLED}
+
+ARG NEWRELIC_LICENSE
```
Comment (sergeyspryker) [inline infrastructure/sdk.sprykerci.Dockerfile:25]: i'm sure that args NEWRELIC_LICENSE NEWRELIC_APPNAME NEWRELIC_LOGLEVEL should be in image. If we push this image the private data be pushed as well

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -14,6 +14,10 @@ services:
       args:
         USER_UID: ${USER_UID}
         SDK_VERSION: ${SDK_VERSION:-latest}
+        NEWRELIC_ENABLED: ${NEWRELIC_ENABLED}
+        NEWRELIC_LICENSE: ${NEWRELIC_LICENSE}
```
Comment (sergeyspryker) [inline docker-compose.dev.yml:18]: I guess it should be moved to env variables (NEWRELIC_LICENSE, NEWRELIC_APPNAME, NEWRELIC_LOGLEVEL)

**Review by sergeyspryker — CHANGES_REQUESTED** [review]

File: docker-compose.dev.yml [hunk 3]
```diff
@@ -14,6 +14,10 @@ services:
       args:
         USER_UID: ${USER_UID}
         SDK_VERSION: ${SDK_VERSION:-latest}
+        NEWRELIC_ENABLED: ${NEWRELIC_ENABLED}
```
Comment (sergeyspryker) [inline docker-compose.dev.yml:17]: Maybe make false by default for more easy dev installation

**Review by sergeyspryker — CHANGES_REQUESTED** [review]

File: docker-compose.dev.yml (hunk 2 above)
Comment (pavelmaksimov25) [reply docker-compose.dev.yml]: thanks, fixed

File: docker-compose.dev.yml (hunk 3 above)
Comment (pavelmaksimov25) [reply docker-compose.dev.yml]: it is an obolete code. removed

**Review by sergeyspryker — APPROVED** [review]

**Pull request merged**